docs-src/clusters/how-to-integrate-elasticsearch-into-a-temporal-cluster.md

```diff
@@ -8,19 +8,25 @@ tags:
   - filtered-lists
   - visibility
 ssdi:
-- Elasticsearch v8 is supported beginning with Temporal Server version 1.18.0
-- Elasticsearch v7.10 is supported beginning with Temporal Server version 1.17.0
-- Elasticsearch v6.8 is supported through Temporal Server version 1.17.x
-- Elasticsearch v6.8 and v7.10 are explicitly supported with AWS Elasticsearch
+  - Elasticsearch v8 is supported beginning with Temporal Server version 1.18.0
+  - Elasticsearch v7.10 is supported beginning with Temporal Server version 1.17.0
+  - Elasticsearch v6.8 is supported through Temporal Server version 1.17.x
+  - Elasticsearch v6.8 and v7.10 are explicitly supported with AWS Elasticsearch
 ---
 
-[Advanced Visibility](/concepts/what-is-advanced-visibility) features depend on an integration with Elasticsearch.
+You can set up [Advanced Visibility](/concepts/what-is-advanced-visibility) by integrating your Temporal Cluster with ElasticSearch.
```

Comment:
We need to nix the term "Advanced Visibility" altogether.
We just have "Visibility" now - which includes using Search Attributes.

Also I think in this node we would be coming at it from the - "You can use Elasticsearch as a dedicated Visibility data store. This is recommended for large scale Clusters and is required when using Cassandra for normal persistence" 

Link to how to set that up with MySQL -